packages/pn-personafisica-webapp/src/component/Contacts/CourtesyContactItem.tsx

```diff
@@ -125,14 +134,17 @@ const CourtesyContactItem = ({ recipientId, type, value, blockDelete }: Props) =
           onConfirmClick={handleEditConfirm}
           blockDelete={blockDelete}
           forceMobileView
+          resetModifyValue={resetModifyValue}
```

Comment:
The resetModifyValue can be replaced by () => handleEditConfirm('cancelled')

Reply:
do it

Reply:
**Beware**  
don't forget to transcribe all the modifications indicated by @AndreaCimini90 unto the "twin" components in pn-personagiuridica-webapp.